src/tv2-common/cues/dve.ts

```diff
@@ -71,5 +71,5 @@ export function TemplateIsValid(templateRaw: string): boolean {
 }
 
 export function GetDVETemplate(config: DVEConfigInput[], templateName: string): DVEConfigInput | undefined {
-	return config ? config.find(c => c.DVEName.toString().toUpperCase() === templateName.toUpperCase()) : undefined
+	return config ? config.find((c) => c.DVEName.toString().toUpperCase() === templateName.toUpperCase()) : undefined
```

Comment:
In what scenarie is a undefined given as the first argument? 
That appears to be the only reason for the undefined check, and in the end it just passes the undefined check further along the chain. 
It does prevent a `TypeError: Cannot read properties of undefined (reading 'find')`, here.

If an empty array is given as the first argument instead, the `.find` will always return undefined, instead of the ternary' undefined. Perhabs updating the boolean to check for empty could lower the the processing power used a smidge.
`return config && config.length !== 0 ? ...`

Reply:
Hopefully, in no scenarios. 
You don't even check for the length. As you point out yourself `.find()` return undefined if the length is zero/it can't find a match. So `return config.find(/* your predicate */)` will give the exact same result